utils/to_string.hh

```diff
@@ -40,6 +42,8 @@ sstring join(sstring delimiter, const PrintableRange& items) {
     return join(delimiter, items.begin(), items.end());
 }
 
+namespace internal {
```

Comment:
Why does Scylla need the utils::internal namespace? It's not really a general-purpose library, so everything in it is "internal", isn't it?